Formula/r/rnp.rb

```diff
@@ -0,0 +1,32 @@
+class Rnp < Formula
+  desc "High performance C++ OpenPGP library used by Mozilla Thunderbird"
+  homepage "https://github.com/rnpgp/rnp"
+  url "https://github.com/rnpgp/rnp/releases/download/v0.18.0/rnp-v0.18.0.tar.gz"
+  sha256 "a90e3ac5b185a149665147f9284c0201a78431e81924883899244522fd3f9240"
+  license all_of: ["MIT", "BSD-2-Clause", "BSD-3-Clause"]
+  head "https://github.com/rnpgp/rnp.git", branch: "main"
+
+  depends_on "cmake" => :build
+  depends_on "botan"
+  depends_on "json-c"
+  uses_from_macos "bzip2"
+  uses_from_macos "zlib"
+
+  def install
+    system "cmake", "-S", ".", "-B", "build",
+           "-DBUILD_TESTING=OFF", *std_cmake_args
```

Comment:
build_testing is already off by default

https://github.com/Homebrew/brew/blob/master/Library/Homebrew/formula.rb#L1795